### PR TITLE
feature/BDD-GWT-IP2-85-list-users by-role

### PR DESCRIPTION
**Pull request by AdamAndrysek**

### CODE REVIEW

File: RestSharpProject/Features/ListUsersByRole.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿Feature: Listing users by given role
+	User Story: There will be a user story when it's ready
+	Task in Java Board: https://tracker.intive.com/jira/browse/IP2-85
+	Task in QA Board: https://tracker.intive.com/jira/browse/IP2-190
+
+Background: 
+Given URL endpoint is /api/users
+
+@mytag
```
Comment (katkoch) [inline RestSharpProject/Features/ListUsersByRole.feature:9]: @my tag is not necessary unless you want to give it function

File: RestSharpProject/Features/ListUsersByRole.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿Feature: Listing users by given role
+	User Story: There will be a user story when it's ready
+	Task in Java Board: https://tracker.intive.com/jira/browse/IP2-85
+	Task in QA Board: https://tracker.intive.com/jira/browse/IP2-190
+
+Background: 
+Given URL endpoint is /api/users
```
Comment (katkoch) [inline RestSharpProject/Features/ListUsersByRole.feature:7]: URL is not necessary.

**Review by katkoch — COMMENTED** [review]
How do you feel about creating two tests? One for leader and one for candidate. Then in line 17 you could be more specific about whose data you are getting.

File: RestSharpProject/Features/ListUsersByRole.feature (hunk 1 above)
Comment (AdamAndrysek) [reply RestSharpProject/Features/ListUsersByRole.feature:9]: OK, deleted

File: RestSharpProject/Features/ListUsersByRole.feature (hunk 2 above)
Comment (AdamAndrysek) [reply RestSharpProject/Features/ListUsersByRole.feature:7]: Done.

**Comment by AdamAndrysek** [comment]
@katkoch For now I'll stay with one test because in my opinion now in one scenario it checks both list users by LEADER role and by CANDIDATE role – one more scenario will be redundant.

**Review by kmiecikmarcin — APPROVED** [review]

**Review by katkoch — APPROVED** [review]

**Pull request merged**